### PR TITLE
Clean up function spacing

### PR DESCRIPTION
**Pull request by they-them**
I have cleaned up some of the function spacing in the code and changed a few instances of `variable != literal` to `literal != variable` to be more consistent.

### CODE REVIEW

**Comment by they-them** [comment]
Hi there! Just checking in on this pull request? :-) Can it be merged?

File: Sources/Nimble/Expectation.swift [hunk 1]
```diff
@@ -6,7 +6,7 @@ internal func expressionDoesNotMatch<T, U>(_ expression: Expression<T>, matcher:
     msg.to = toNot
     do {
         let pass = try matcher.doesNotMatch(expression, failureMessage: msg)
-        if msg.actualValue == "" {
+        if "" == msg.actualValue {
```
Comment (ikesyo) [inline Sources/Nimble/Expectation.swift:9]: I believe Yoda conditions is not appropriate for Swift: https://realm.github.io/SwiftLint/yoda_condition.html

File: Sources/Nimble/Matchers/Contain.swift [hunk 2]
```diff
@@ -11,7 +11,6 @@ public func contain<S: Sequence>(_ items: S.Element...) -> Predicate<S> where S.
 public func contain<S: Sequence>(_ items: [S.Element]) -> Predicate<S> where S.Element: Equatable {
     return Predicate.simple("contain <\(arrayAsString(items))>") { actualExpression in
         guard let actual = try actualExpression.evaluate() else { return .fail }
-
```
Comment (ikesyo) [inline Sources/Nimble/Matchers/Contain.swift:14]: These are intentional empty lines for making early exits clear.

**Review by ikesyo — CHANGES_REQUESTED** [review]

**Comment by ikesyo** [comment]
The forked repository has gone. Closing.

**Pull request closed without merging**